examples/camera.rs

```diff
@@ -1,6 +1,14 @@
 use macroquad::*;
 
-#[macroquad::main("Camera")]
+fn window_conf() -> Conf {
```

Comment:
It is not critical and I could merge it like this, however, maybe introduce new example just for window config and leave old example with simple no-config main?